src/wallet/feebumper.cpp

```diff
@@ -70,7 +70,7 @@ BumpFeeResult PreconditionChecks(const CWallet* pWallet, const CWalletTx& wtx, s
 
 bool TransactionCanBeBumped(CWallet* pWallet, const uint256& txid)
 {
-    LOCK2(cs_main, pWallet->cs_wallet);
+    LOCK(pWallet->cs_wallet);
```

Comment:
Can you document the exact locking requirements in GetWalletTx in wallet.h or wallet.cpp just to make sure someone doesnt do some crazy-ass change in the future and blow up the lockorder?